Logger/LogFactory.cs

```diff
@@ -2,10 +2,21 @@
 {
     public class LogFactory
     {
+        public string filePath;
+
         public BaseLogger CreateLogger(string className)
         {
-            
-            return null;
+            BaseLogger bLogger = new FileLogger(className,filePath);
```

Comment:
The name property was supposed to be set with an object initializer.
```suggestion
            BaseLogger bLogger = new FileLogger(filePath){ Name = className };
```